lib/fibonacci.rb

```diff
@@ -5,5 +5,21 @@
 # Hint, you may want a recursive helper method
 # Hint:  Fib(0) = 0, Fib(1) = 1, work up from there
 def fibonacci(n)
```

Comment:
This works, but can you do it without maintaining the array of all the fib numbers from 0..n?